docs/tech-reference/iso-rsync-distributor.rst

```diff
@@ -9,6 +9,13 @@ ssh to perform the file transfer.
 
 Configuration
 =============
+Pulp's SELinux policy includes a ``pulp_manage_rsync`` boolean. When enabled, the
+``pulp_manage_rsync`` boolean allows Pulp to execute ssh, execute rsync, and manage sockets. The
```

Comment:
I suggest making this a little less specific since Refpol actually allows a lot of things underneath and is always changing. I recommend this:

s/execute ssh, execute rsync, and manage sockets/use rsync and make ssh connections/